packages/cli/src/commands/doctor/healthchecks/androidSDK.ts

```diff
@@ -61,13 +82,88 @@ export default {
       needsToBeFixed: !isRequiredVersionInstalled,
     };
   },
-  runAutomaticFix: async ({loader, environmentInfo}) => {
-    const version = environmentInfo.SDKs['Android SDK'];
-    const isSDKInstalled = version !== 'Not Found';
+  win32AutomaticFix: async ({loader}) => {
+    // TODO: Create a GitHub action that checks and updates this periodically?
+    const cliToolsUrl =
+      'https://dl.google.com/android/repository/commandlinetools-win-6200805_latest.zip';
+
+    const systemImage = 'system-images;android-28;google_apis;x86_64';
+    // Installing 29 as well so Android Studio does not complain on first boot
+    const componentsToInstall = [
+      'platform-tools',
+      'build-tools;29.0.3',
+      'platforms;android-29',
+      // Is 28 still needed?
+      'build-tools;28.0.3',
+      'platforms;android-28',
+      'emulator',
+      systemImage,
+      '--licenses', // Accept any pending licenses at the end
+    ];
+
+    const androidSDKRoot = getAndroidSdkRootInstallation();
+
+    await downloadAndUnzip({
+      loader,
+      downloadUrl: cliToolsUrl,
+      component: 'Android Command Line Tools',
+      installPath: androidSDKRoot,
+    });
+
+    for (const component of componentsToInstall) {
+      loader.text = `Installing "${component}" (this may take a few minutes)`;
+
+      try {
+        await installComponent(component, androidSDKRoot);
+      } catch (e) {
+        // Is there a way to persist a line in loader and continue the execution?
+      }
+    }
+
+    loader.text = 'Updating environment variables';
+
+    // Required for the emulator to work from the CLI
+    await setEnvironment('ANDROID_SDK_ROOT', androidSDKRoot);
+    await setEnvironment('ANDROID_HOME', androidSDKRoot);
+    await updateEnvironment('PATH', path.join(androidSDKRoot, 'tools'));
+    await updateEnvironment(
+      'PATH',
+      path.join(androidSDKRoot, 'platform-tools'),
+    );
+
+    loader.text =
+      'Configuring Hypervisor for faster emulation, this might prompt UAC';
+
+    const {hypervisor, installed} = await getBestHypervisor(androidSDKRoot);
+
+    if (!installed) {
+      if (hypervisor === 'none') {
+        loader.warn(
+          'Android SDK configured but virtualization could not be enabled.',
+        );
+        return;
+      }
 
+      if (hypervisor === 'AMDH') {
+        await enableAMDH(androidSDKRoot);
+      } else if (hypervisor === 'HAXM') {
+        await enableHAXM(androidSDKRoot);
+      } else if (hypervisor === 'WHPX') {
+        await enableWHPX();
+      }
+    }
+
+    loader.text = 'Creating AVD';
+    await createAVD(androidSDKRoot, 'pixel_9.0', 'pixel', systemImage);
+
+    loader.succeed(
+      'Android SDK configured. You might have to start a new shell for things to work properly.',
```

Comment:
Perhaps a more direct message will lead to an overall better outcome? E.g.

```suggestion
      'Android SDK configured. Start a new shell for the configuration to take effect.',
```